_extensions/onlimit-theme/_brand.yml

```
@@ -0,0 +1,39 @@
+meta:
+  name:
+    full: ON-LiMiT Theme
+    short: onlimit-theme
+  link:
+    github: https://github.com/onlimit-study/onlimit-theme
```

```suggestion
    full: ON-LiMiT Study
    short: onlimit
  link:
    github: https://onlimit.org
```

Looking at https://posit-dev.github.io/brand-yml/brand/meta.html it seems that we would use this to link to the "brand" itself, as in, ON-LiMiT. Maybe include a link to that instead?